exercises/grains/src/example.h

```diff
@@ -0,0 +1,7 @@
+#ifndef _GRAINS_H
+#define _GRAINS_H
+
+unsigned long square(int sq);
+unsigned long total(void);
```

Comment:
@ryanplusplus, I'm assuming you mean switch over to stdint types in the entire exercise, right?